### PR TITLE
Deleting temporary files and directories in tests

### PR DESCRIPTION
**Pull request by yfarjoun**

### CODE REVIEW

File: src/tests/java/picard/illumina/IlluminaBasecallsToFastqTest.java [hunk 1]
```diff
@@ -127,6 +128,7 @@ public void testDualBarcodes() throws Exception {
      * @param readStructureString
      * @throws Exception
      */
+
```
Comment (mattsooknah) [inline src/tests/java/picard/illumina/IlluminaBasecallsToFastqTest.java]: Could you update the ```@param```s in the doc?

**Comment by mattsooknah** [comment]
One tiny comment, otherwise :+1:

**Comment by yfarjoun** [comment]
sorry this took so long. back to you @mattsooknah

**Comment by mattsooknah** [comment]
:+1:

**Pull request merged**